sdk/eventhub/event-hubs/package.json

```diff
@@ -29,18 +29,18 @@
     "dist/",
     "dist-esm/src/",
     "src/",
-    "typings/src",
+    "types/src",
     "tsconfig.json"
   ],
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
-    "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
-    "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
+    "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1 && npm run extract-api",
+    "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1 && npm run extract-api",
```

Comment:
Will the browser vs node produce different results?

Reply:
Great question. Do we put the browser .d.ts in a different location?

Reply:
The input to the api extractor is `typings/src/index.d.ts`
The d.ts file is created when we run `tsc -p .` which doesnt care about node vs browser as far as I know.
Its the rollup process that creates different output for node vs browser.

So, I would suggest to add the extract step just to the `build` script. I believe this is the script that is run by CI as well.